src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -170,6 +174,15 @@ public static void AddCommonServices(IServiceCollection services)
             // specialized app.
             services.AddSingleton<IHostIdProvider, ScriptHostIdProvider>();
             services.TryAddSingleton<IScriptEventManager, ScriptEventManager>();
+
+            // Add Language Worker Services
+            // Need to maintain the order: Add RpcServerInitializationService and then LanguageWorkerChannelInitializationService
+            services.AddSingleton<IHostedService, RpcServerInitializationService>();
+            services.AddSingleton<IHostedService, LanguageWorkerChannelInitializationService>();
```

Comment:
@fabiocav / @paulbatum  - Removing LanguageWorkerChannelInitializationService does not start java worker at webhost level